lib/CSSStyleDeclaration.js

```diff
@@ -298,351 +180,354 @@ class CSSStyleDeclaration {
   }
 
   /**
-   * @param {...number} args
+   * Returns the priority of the property (e.g. "important").
+   *
+   * @param {string} property
+   * @returns {string}
    */
-  item(...args) {
-    if (!args.length) {
-      const msg = "1 argument required, but only 0 present.";
-      throw new this._global.TypeError(msg);
+  getPropertyPriority(property) {
+    return this._priorities.get(property) || "";
+  }
+
+  /**
+   * Sets a property with a value and optional priority.
+   *
+   * @param {string} prop - Property name
+   * @param {string} val - Property value
+   * @param {string} [prior] - Priority (e.g. "important")
+   */
+  setProperty(prop, val, prior) {
```

Comment:
Please replace "prior" with "priority"; they are two different words that mean very different things.